docs/src/components/banner/AppHeaderBanner.tsx

```diff
@@ -19,7 +19,7 @@ function getSurveyMessage() {
           },
         }}
       >
-        MUI Developer survey 2022 →
+        MUI Developer survey 2023 →
```

Comment:
```suggestion
        2023 MUI developer survey →
```

We always put the year first!

<img width="400" alt="Screenshot 2024-01-12 at 08 27 04" src="https://github.com/mui/material-ui/assets/67129314/5d8dc8e7-eb67-4ed7-9d81-0d238c5dae05">